cluster/charts/rook-ceph/templates/NOTES.txt

```diff
@@ -1,5 +1,18 @@
 The Rook Operator has been installed. Check its status by running:
   kubectl --namespace {{ .Release.Namespace }} get pods -l "app=rook-ceph-operator"
 
-Visit https://rook.io/docs/rook/master for instructions on how
-to create & configure Rook clusters
+Visit https://rook.io/docs/rook/master for instructions on how to create and configure Rook clusters
```

Comment:
Please use `master` as the version in this link and the others, as this is going to be merged into `master` first (on backport to v0.9 this would be changed to v0.9 accordingly) and the first important note bulletin point matches with all links in this message.